src/sage/combinat/crystals/alcove_path.py

```diff
@@ -1040,12 +1040,13 @@ def f(self, i):
         #
         # otherwise if m_index - 1 > 0 then (C2) is enough
 
-        if ( (not finite_cartan_type or i != 0) and M > 0  # alpha_i is a simple root
-           ) or KR_test :# KR case
+        if ((not finite_cartan_type or i != 0) and M > 0  # alpha_i is a simple root
+            ) or KR_test:  # KR case
```

Comment:
I don't get the alignment here. the `)` is closing `((not finite_cartan_type or i != 0) and M > 0'.
In fact, it's weird to have this parenthesis on a new line...